ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/GetJobManagement_Test.java

```diff
@@ -178,6 +180,7 @@ public void createReadJobWithBigFile() throws IOException, URISyntaxException, N
 
         } finally {
             FileUtils.deleteDirectory(tempDirectory.toFile());
+            deleteBigFile();
```

Comment:
This will delete the file from BUCKET_NAME, but wasn't the file placed in DIR_NAME in this test?